extension/popup.html

```diff
@@ -28,6 +28,15 @@ <h2 class="subtitle">
         <i class="material-icons">pause</i>
       </button>
     </div>
+    <div class="volumebox">
+      <span class="icon">
+        <i class="material-icons">volume_down</i>
+      </span>
+      <progress id="volume" class="progress is-primary" value="100" max="100"></progress>
```

Comment:
Why `progress` over an `input` of type `range`? http://www.html5tutorial.info/html5-range.php